.github/workflows/release.yml

```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   release:
     runs-on: ubuntu-latest
-    name: A job to execute release
+    name: Release top npm
```

Comment:
I don't really understand this description tbh :D

Reply:
Top wants to be to :D Ty